### PR TITLE
[RELEASE] connects Falkonry/tracker#2193

### PR DESCRIPTION
**Pull request by aniket-amrutkar**

### CODE REVIEW

File: test/TestAssessmentGetOutput.py [hunk 1]
```diff
@@ -26,6 +26,23 @@ def test_get_assessment_output(self):
             print(exception_handler(e))
             self.assertEqual(0, 1, 'Error getting output of a Assessment')
 
+    @unittest.skip("streaming can only be done once ")
+    def test_get_assessment_output_with_offset(self):
```
Comment (phagunbaya) [inline test/TestAssessmentGetOutput.py]: Inappropriate example. Should explain where to get the offset from and then how to set it.

File: falkonryclient/service/falkonry.py [hunk 2]
```diff
@@ -309,6 +309,10 @@ def get_output(self, assessment, options):
             options['format'] = None
 
         url = '/assessment/' + str(assessment) + '/output'
+
+        if 'offset' in options:
```
Comment (phagunbaya) [inline falkonryclient/service/falkonry.py]: add options to the param doc

File: test/TestAssessmentGetOutput.py [hunk 3]
```diff
@@ -26,6 +27,42 @@ def test_get_assessment_output(self):
...
+        fclient = FClient(host=host, token=token,options=None)
+
+        try:
+            stream = fclient.get_output(assessment, {})
+            lastOffset = 0
+            for event in stream.events():
+                print(json.dumps(json.loads(event.data)))
+                lastOffset = json.loads(event.data)['offset'] # keep track of offset sent in falkonry output event
+
+        except Exception as e:
```
Comment (jeegs6up) [inline test/TestAssessmentGetOutput.py]: Are we blanketing all exceptions? What happens when live streaming is turned off  - won't that cause an exception here?

Comment (phagunbaya) [reply test/TestAssessmentGetOutput.py]: It will and the exception will have the human-readable message.

**Pull request merged**